packages/devtools_app/lib/src/ui/service_extension_widgets.dart

```diff
@@ -508,6 +509,241 @@ class _ServiceExtensionCheckboxState extends State<ServiceExtensionCheckbox>
   }
 }
 
+/// A button that, when pressed, will display an overlay directly below that has
+/// a list of service extension checkbox settings.
+class ServiceExtensionCheckboxOverlayButton extends StatefulWidget {
```

Comment:
The name doesn't capture that it is for multiple service extensions rather than one.

How about ServiceExtensionCheckboxGroupButton ?